### DISCUSSION TITLE
Mapping 1 front port to multiple rear ports

### DISCUSSION THREAD

**Discussion by dpotter-intoto** [netbox-community/netbox#13187] [Q&A]
Any recommendations on how to model a patch panel with a small number of connectors (e.g., 4x MPO-12) on the front panel and a larger number of connectors (e.g., 24x Duplex LC) on the rear?

Do I need to "turn around" the panel and reverse the front/rear connectors to achieve this?

**Comment by candlerb** [comment]
Yes. The names "FrontPort" and "RearPort" only refer to the typical usage. Netbox doesn't even know whether the ports are *actually* on the front or rear of the device.
